idb/postgres/postgres.go

```diff
@@ -175,6 +175,12 @@ func (db *IndexerDb) init(opts idb.IndexerDbOptions) (chan struct{}, error) {
 
 // AddBlock is part of idb.IndexerDb.
 func (db *IndexerDb) AddBlock(vb *itypes.ValidatedBlock) error {
+	protoVersion := protocol.ConsensusVersion(vb.Block.CurrentProtocol)
+	_, ok := config.Consensus[protoVersion]
```

Comment:
this reminds me of `EnableAssetCloseAmount`. do we still need to support `AssetCloseAmount` for the older txns? maybe it's one of the things we can deprecate in the next release?